io-package.json

```diff
@@ -4,8 +4,20 @@
         "require": {
             "js-controller": ">0.6.6"
         },
-        "version": "1.1.1",
+        "version": "1.2.2",
```

Comment:
Because 1.2.0/1/2 was never available on main repo and used by users it would be cleaner to only have a v1.2.0 with all your changes ... ?!

Reply:
1.2.0/1 were test versions. Is ok. :9